package.json

```diff
@@ -43,7 +43,7 @@
     "build:resources": "parcel build src/resources/**/*.html src/resources/**/**/*.html --no-source-maps",
     "build:squiz": "parcel build src/index.html --no-source-maps",
     "svg-icons-to-png": "convert-svg-to-png --height 24px --width 24px src/assets/*.svg && mv src/assets/*.png static",
-    "postinstall": "rm -rf ./.cache && node ./postinstall.js",
+    "postinstall": "node ./postinstall.js",
```

Comment:
why remove the cleanup of .cache?

Reply:
Because the code you have supplied "only" works on a mac and we are not solely based on this OS